presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleConfig.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import io.airlift.configuration.Config;
+import io.prestosql.plugin.jdbc.BaseJdbcConfig;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.math.RoundingMode;
+
+public class OracleConfig
+        extends BaseJdbcConfig
+{
+    private boolean includeSynonyms;
+    private int fetchSize = 1000;
+    private int varcharMaxSize = 4000;
+    private int timestampDefaultPrecision = 6;
+    private int numberDefaultScale = 10;
+    private RoundingMode numberRoundingMode = RoundingMode.HALF_UP;
+
+    public int getFetchSize()
+    {
+        return fetchSize;
+    }
+
+    @Config("oracle.fetch-size")
+    public OracleConfig setFetchSize(int fetchSize)
+    {
+        this.fetchSize = fetchSize;
+        return this;
+    }
+
+    @NotNull
+    public boolean isIncludeSynonyms()
+    {
+        return includeSynonyms;
+    }
+
+    @Config("oracle.include-synonyms")
```

Comment:
Most existing properties for boolean use `xxx.enabled` format. Could you rename to `oracle.synonyms.enalbed`?

Reply:
done